charts/div-ccd-definitions/values.preview.template.yaml

```diff
@@ -136,6 +137,7 @@ ccd:
         PROXY_DOCUMENT_MANAGEMENT: http://dm-store-aat.service.core-compute-aat.internal
         PROXY_PRINT_SERVICE: http://ccd-case-print-service-${SERVICE_FQDN}
         ADDRESS_LOOKUP_TOKEN: ${ADDRESS_LOOKUP_TOKEN}
+        PROXY_DATA: http://ccd-data-store-api-aat.service.core-compute-aat.internal
```

Comment:
I have had problems testing this on another PR. Cases from aat are found successfully. The PR specific ccd spreadsheet is not being used. It is probably using the aat one instead. The solution for this needs to be found otherwise it will prevent pod testing.

Reply:
I'll have a look, an alternative would be to create a separate COS PR and point AAC to definition's PR data store.